simulation/src/test/resources/txt/server_side_rate_limits_with_sticky_clients_steady_vs_bursty_client[CONCURRENCY_LIMITER_ROUND_ROBIN].txt

```diff
@@ -1,3 +1,3 @@
-success=97.0%	client_mean=PT24.931845968S	server_cpu=PT1M37.525S    	client_received=10060/10060	server_resps=19505	codes={200=9755, 429=305}
-client=slowAndSteady endpoint	client_mean=PT14.795458286S
-client=oneShotBurst endpoint	client_mean=PT24.992664294S
+success=96.9%	client_mean=PT24.927819953S	server_cpu=PT1M37.475S    	client_received=10060/10060	server_resps=19495	codes={200=9749, 429=311}
+client=slowAndSteady endpoint	client_mean=PT0.450916666S 
+client=oneShotBurst endpoint	client_mean=PT25.074681373S
```

Comment:
`slowAndSteady` shows significant improvement while `onsShotBurst` is only slightly slower, this is what we're looking for.